### PR TITLE
Chapter Image settings(front-end)

### PR DESCRIPTION
**Pull request by negulici-r-barnabas**
Chapter Image (Dummy Chapters) can be configured from the front-end by the user.
There are 3 settings:
- duration (seconds)
- count
- resolution (16:9 aspect ratio)

The extraction duration (interval) now can be set in seconds. The minimal value being 1 second. The default value is "300" (5 minutes).
The count is the image file limit / media file, just in case there is bad metadata. Default value is "100".
The resolution sets the extracted image size. It ranges from 144p to 2160p (16:9 aspect ratio). Default value: "Match Source". 

Screenshot of the Dashboard > General settings page:
Screenshot 2022-07-18 175753

Screenshot of the resolution dropdown:

Screenshot 2022-07-18 175845

I tried to keep the changes to minimal. I just used the server configuration that was already implemented by adding 3 more properties to its model class.

I'm a beginner coder. I welcome any improvement suggestions.

The issues that would resolve is  [#1346](https://github.com/jellyfin/jellyfin/issues/1346) (already closed but not implemented).
The feature request is [here](https://features.jellyfin.org/posts/16/allow-custom-chapter-image-intervall).

This is the front-end implementation. The back-end implementation is [here](https://github.com/jellyfin/jellyfin/pull/8137).

### CODE REVIEW

**Comment by HerrEiklar** [comment]
Status on this?

**Comment by thornbill** [comment]
Web changes are blocked until the server changes are reviewed + merged.

**Comment by negulici-r-barnabas** [comment]
Hello, sorry for my absence I have modified my code with the requested changes on the back-end and I also added 144p to the list of selectable resolutions.

**Review by craigers521 — APPROVED** [review]

File: src/controllers/dashboard/general.js
Comment (thornbill) [inline src/controllers/dashboard/general.js:27]: If you run `npx eslint --fix` it should fix the styling issues that are listed here.

**Comment by koosc** [comment]
If fixing the linting the only thing pending? I'd be happy to fix that if that's all it is waiting on.

**Review by nielsvanvelzen — APPROVED** [review]

**Comment by Deagan** [comment]
I can't seem to find these settings? Where can I edit these settings on v10.8.9?

**Pull request merged**
